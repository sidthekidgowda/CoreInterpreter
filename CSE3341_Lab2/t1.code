
program int X , Y , XY 	;

          begin input X ; Y  := 0 ;
     
               do 

                              input XY ; output XY ;

                              Y  := Y + XY ; X  := X - 1 ;

			while [ X > 0 ]

                          enddo ;   

	       output Y ;

          end  
      
