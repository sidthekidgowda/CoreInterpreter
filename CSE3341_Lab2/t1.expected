program
  int X,Y,XY;
begin
  input X;
  Y:=0;
  do
    input XY;
    output XY;
    Y:=Y+XY;
    X:=X-1;
  while[X>0]enddo;
  output Y;
end
7
-4
10
5
-9
9
